xrootd/xrdproto/closereq/closereq.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package closereq contains the structures describing request and response for close request.
+// See xrootd protocol specification (http://xrootd.org/doc/dev45/XRdv310.pdf, p. 41) for details.
+package closereq // import "go-hep.org/x/hep/xrootd/xrdproto/closereq"
```

Comment:
`closereq` reads funny when you ask for a `Response` (and stutters somewhat with `Request`.)

naming is hard but here are some possibilities:
- `closexrd`, `xrdclose`
- `closeop`, `opclose`